meal-planning/src/components/shopping-list/ShoppingList.js

```diff
@@ -42,7 +42,7 @@ const ShoppingList = () => {
         <Button size="small" onClick={openModal} endIcon={<AddCircleOutline />}>
           Add
         </Button>
-        <AddIngredientDialog
+        <RecipeDialog
           isOpen={isModalOpen}
           handleDialogSubmit={handleDialogSubmit}
```

Comment:
there is a bug with this on close, I think you forgot to update some stuff

Reply:
restored AddIngredientDialog component